packages/atomic-react/src/components/SearchInterfaceWrapper.tsx

```diff
@@ -24,6 +25,14 @@ interface WrapperProps extends Omit<JSX.AtomicSearchInterface, 'i18n'> {
    *
    */
   localization?: (i18n: i18n) => void;
+  /**
+   * @deprecated this option should instead be part of the `engine`'s `search` configuration.
```

Comment:
It could say something like "this option as no effect" as well

Reply:
Good idea JF
```suggestion
   * @deprecated This option has no effect. Rather, set the pipeline through the `engine` `search` configuration.
```